README.md

```diff
@@ -1,28 +1,32 @@
 # FizzySteamyMirror
 
-This is a community maintained repo forked from [RayStorm](https://github.com/Chykary/FizzySteamyMirror). Mirror [docs](https://mirror-networking.com/docs/Transports/Fizzy.html).
+This is a community maintained repo forked from **[RayStorm](https://github.com/Chykary/FizzySteamyMirror)**. 
 
-FizzySteamyMirror brings together [Steam](https://store.steampowered.com/) and [Mirror](https://github.com/vis2k/Mirror) utilising Async of a Steam P2P network transport layer for **Mirror**.
+Mirror **[docs](https://mirror-networking.com/docs/Transports/Fizzy.html)** and the official community **[Discord](https://discord.gg/N9QVxbM)**.
+
+FizzySteamyMirror brings together **[Steam](https://store.steampowered.com)** and **[Mirror](https://github.com/vis2k/Mirror)** utilising Async of a Steam P2P network transport layer for **Mirror**.
 
 ## Dependencies
-**Note: SteamWorks.NET is packed with this transport.**
+If you want an easy import, skip the steps bellow & download the latest **[unitypackage](https://github.com/Chykary/FizzySteamyMirror/releases)**, **it has SteamWorks.Net already included**.
+
+**Note: If you already have SteamWorks.Net in your project, you might need to delete either your import or the one included in the FizzySteamyMirror [unitypackage](https://github.com/Chykary/FizzySteamyMirror/releases).**

```

Comment:
Some people was installing the **unitypackage** while also they had **SteamWorks.Net** imported, which then created conflicts.